jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -49,8 +57,60 @@
 /*[ENDIF]*/
 			PlatformLoggingMXBean {
 
-	private static final LoggingMXBeanImpl instance = new LoggingMXBeanImpl();
+	private static final LoggingMXBeanImpl instance;
+
+/*[IF Sidecar19-SE]
+	private static final Module java_logging;
+/*[ENDIF]*/
+	private static final Class<?> logManagerClass;
+	private static final Class<?> loggerClass;
+	private static final Class<?> levelClass;
+	private static final Method logManager_getLogManager;
+	private static final Method logManager_getLogger;
+	private static final Method logManager_getLoggerNames;
+	private static final Method logger_getLevel;
+	private static final Method logger_getParent;
+	private static final Method logger_getName;
+	private static final Method logger_setLevel;
+	private static final Method level_getName;
+	private static final Method level_parse;
+	private static final String logManager_LOGGING_MXBEAN_NAME;
+	
+	static {
+		try {
+/*[IF Sidecar19-SE]
+/*[IF Sidecar19-SE-B165]
+			java_logging = ModuleLayer.boot().findModule("java.logging").get();
+/*[ELSE]
+			java_logging = Layer.boot().findModule("java.logging").get();
```

Comment:
This can fail if the (optional) java.logging module is not available, but we still need to provide an implementation of `PlatformLoggingMXBean`.

Reply:
The reason I did it like this is because the constructor of this class is dependent on `logManager_LOGGING_MXBEAN_NAME` being set properly. I could set `objectName` conditionally on `java.logging` being found.